public/static/pathways/her2_pathway.json

```diff
@@ -0,0 +1,206 @@
+{
+  "name": "Early Stage HER2+ Pathway (draft)",
```

Comment:
`draft` should be capitalized
![Screen Shot 2020-02-27 at 8 10 25 AM](https://user-images.githubusercontent.com/41651655/75448197-bd77b100-5938-11ea-866f-c49f01ee6261.png)